doc/docusaurus/docs/delve-deeper/haddock-documentation.md

```diff
@@ -2,16 +2,11 @@
 sidebar_position: 3
 ---
 
-# Haddock documentation
+# Haddock Documentation
 
-## Public Plutus code libraries
+Haddock is a tool for automatically generating documentation from annotated Haskell source code.
+The Haddock for the latest release of public libraries from the [plutus repository](https://github.com/IntersectMBO/plutus) is available [here](https://plutus.cardano.intersectmbo.org/haddock/latest).
+You can also use the dropdown menu to view the Haddock for an older release or the master branch.
```

Comment:
Oh, cool, I wasn't aware of that. It doesn't appear to work for older releases though? Or at least I'm only seeing master, latest and 1.35.0.0.

Reply:
See https://github.com/IntersectMBO/plutus-private/issues/1181